spec/featured-spec.js

```diff
@@ -4,10 +4,6 @@ var yaml = require('yamljs')
 
 test('Limit featured homepage apps to 24', function (t) {
   t.plan(1)
-
-  var apps = yaml.load(path.join(__dirname, '..', '_data', 'apps.yml'))
-  var featuredApps = apps.filter(function (app) {
-    return app.featuredOnHomepage
-  })
+  var featuredApps = require(path.join(__dirname, '../_data/featured_apps.json'))
```

Comment:
Oh really!? You mean the `'../_data/featured_apps` part? I thought `path.join` took care of that. I guess that means I've been writing a lot of windows-unfriendly path code. 🙈